### PR TITLE
Feature/task 1 container model

### PR DESCRIPTION
**Pull request by MetiuOol**

### CODE REVIEW

File: src/main/java/pl/kurs/watercontainers/models/WaterContainer.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import java.util.Objects;
+
+
+public class WaterContainer implements Serializable {
+    static final long serialVersionUID = 1L;
+
+    private final String name;
+    private final double maxCapacity;
+    private double waterLevel;
+
```
Comment (michalsloniewski) [inline src/main/java/pl/kurs/watercontainers/models/WaterContainer.java:17]: Please make this class java beans compatibile (visible default constructor ect.)

**Review by michalsloniewski — CHANGES_REQUESTED** [review]

File: src/main/java/pl/kurs/watercontainers/models/WaterContainer.java (hunk 1 above)
Comment (MetiuOol) [reply src/main/java/pl/kurs/watercontainers/models/WaterContainer.java:17]: Java beans has no sense here, because we have final fields.

Comment (michalsloniewski) [reply src/main/java/pl/kurs/watercontainers/models/WaterContainer.java:17]: all right then

File: src/main/java/pl/kurs/watercontainers/exceptions/InvalidLevelException.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package pl.kurs.watercontainers.exceptions;
+
+public class InvalidLevelException extends RuntimeException {
+    public InvalidLevelException(String message) {
+        super(message);
+    }
+}
```
Comment (michalsloniewski) [inline src/main/java/pl/kurs/watercontainers/exceptions/InvalidLevelException.java:11]: please add constructor with message and cause

File: src/main/java/pl/kurs/watercontainers/exceptions/InvalidCapacityException.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package pl.kurs.watercontainers.exceptions;
+
+public class InvalidCapacityException extends RuntimeException {
+    public InvalidCapacityException(String message) {
+        super(message);
+    }
+}
```
Comment (michalsloniewski) [inline src/main/java/pl/kurs/watercontainers/exceptions/InvalidCapacityException.java:11]: please add constructor with message and cause

**Review by michalsloniewski — APPROVED** [review]

**Pull request merged**